src/main/java/frc/robot/subsystems/base/drivetrain/DrivetrainBase.java

```diff
@@ -0,0 +1,76 @@
+package frc.robot.subsystems.base.drivetrain;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * @author Barel
+ * @version 1.0
+ * @since 2020-off
+ * <p>
+ * The class contains various methods that every {@code Wrist} <b>should</b> implement.
+ * <p>
+ * The purpose of this class is to provide guidelines and restrict what important to us and what not.
+ */
+public abstract class DrivetrainBase extends SubsystemBase {
```

Comment:
Please add the unitModel

Reply:
I don't think that I should add the unit model inside the abstract class, I think it more user choice.

Reply:
Fair enough